Reqnroll/Assist/InstanceCreationOptions.cs

```diff
@@ -3,6 +3,7 @@ namespace Reqnroll.Assist
     public class InstanceCreationOptions
     {
         public bool VerifyAllColumnsBound { get; set; }
+        public bool CaseInsensitiveColumnVerify { get; set; }
```

Comment:
Maybe start with Verify, like the  VerifyAllColumnsBound? (consistency)

e.g.  VerifyCaseInsensitive

Reply:
All other looks good